digits/utils/__init__.py

```diff
@@ -25,7 +25,7 @@ def is_url(url):
 
 def wait_time():
     """Wait a random number of seconds"""
-    return uniform(0.3, 0.5)
+    return uniform(0.05, 0.1)
```

Comment:
We can leave this in for now. I suspect it may cause display issues (pages getting loaded and requesting information from the server which isn't available yet). Just making a note here so I can say "I told you so" later :smile: